### ISSUE TITLE
timeshift v18.6 restore cancel process failed

### ISSUE THREAD

**Issue by heysion** [teejee2008/timeshift#306]
**Describe the bug**
timeshift v18.6 restore cancel process failed

**To Reproduce**
1. ceate a rsync backup
2. Click on restore and select backup items
3. wait the progress bar over 10% , cancel it .

**Expected behavior**
cancel succeed

**Screenshots**
[timeshift cancel failed](http://paste.ubuntu.org.cn/i4358666)

**System (please complete the following information):**
 - Linux Distribution deepin 15.7 like debian 9.x
 - Desktop awesome and deepin dde
 - Application Version v18.6

***Consider making a donation to support this project***

[![](https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif)](https://www.paypal.com/cgi-bin/webscr?business=teejeetech@gmail.com&cmd=_xclick&currency_code=USD&item_name=Timeshift%20Donation)

**Comment by teejee2008** [comment]
Why are you trying to cancel the restore?

Cancelling an rsync restore will leave the target system in an inconsistent state. Please be certain you wish to do the restore before starting it.

**Comment by heysion** [comment]
> Why are you trying to cancel the restore?
> 
> Cancelling an rsync restore will leave the target system in an inconsistent state. Please be certain you wish to do the restore before starting it.

Yeah . I know it . But I found problem was not throw from rsync cancel process . I checked  the  source code which transform  after the valac . Maybe was the issues due to callback gtk window when after rsync cancel process ending .
